### PR TITLE
swagger/fix: []string are not enum by swagger definition

### PR DESCRIPTION
**Pull request by sapk**
`[]string` are not enum by swagger definition this led the generation of a client to think that `repo.code,repo.issues,repo.ext_issues,repo.wiki,repo.pulls,repo.releases,repo.ext_wiki` is an array of `[]string` (`[][]string`).
By definition, enum are one of an array like `Permission` that can take one value from `none,read,write,admin,owner`.
This PR change the definition to an example to be able to generate a valid client from the swagger specs. Currently the generated client doesn't compile because it try convert the `[]string` enum to `[][]string`

### CODE REVIEW

**Comment by sapk** [comment]
Somewhat related (at least for the generate limitation part) : https://github.com/go-swagger/go-swagger/issues/1942

File: modules/structs/org_team.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ type CreateTeamOption struct {
 	Description string `json:"description" binding:"MaxSize(255)"`
 	// enum: read,write,admin
 	Permission string `json:"permission"`
-	// enum: repo.code,repo.issues,repo.ext_issues,repo.wiki,repo.pulls,repo.releases,repo.ext_wiki
+	// example: [repo.code,repo.issues,repo.ext_issues,repo.wiki,repo.pulls,repo.releases,repo.ext_wiki]
```
Comment (guillep2k) [inline modules/structs/org_team.go]: If those are strings, wouldn't it be like...?
```
example: ["repo.code","repo.issues","repo.ext_issues","repo.wiki","repo.pulls","repo.releases","repo.ext_wiki"]
```

Comment (sapk) [reply modules/structs/org_team.go]: done

**Comment by guillep2k** [comment]
image

Is it OK that `units` and `permission` have different representations?

**Comment by sapk** [comment]
Yes because permission only take one of the value (enum type) and units is a list (array) of manyof the value wich is not really an enum.

**Review by guillep2k — APPROVED** [review]

**Comment by sapk** [comment]
This could maybe improved by defining an Go enum type for units and déclare that units is an array of this Type. I have try but doesn't have anythings fully working and it have a lot of refactor to be done on other parts of code. There also seems to be issue whit that on go-swagger so I used the example fields.

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**